core/src/main/java/org/jruby/ext/ffi/Platform.java

```diff
@@ -317,49 +317,12 @@ public static void createPlatformModule(Ruby runtime, RubyModule ffi) {
         OS_TYPE os = platform.getOS();
         module.defineConstant("ADDRESS_SIZE", runtime.newFixnum(platform.addressSize));
         module.defineConstant("LONG_SIZE", runtime.newFixnum(platform.longSize));
-        module.defineConstant("OS", runtime.newString(OS.toString()));
-        module.defineConstant("ARCH", runtime.newString(platform.getCPU().toString()));
-        module.defineConstant("NAME", runtime.newString(platform.getName()));
-        module.defineConstant("IS_WINDOWS", runtime.newBoolean(os == OS.WINDOWS));
-        module.defineConstant("IS_BSD", runtime.newBoolean(platform.isBSD()));
-        module.defineConstant("IS_FREEBSD", runtime.newBoolean(os == OS.FREEBSD));
         module.defineConstant("IS_DRAGONFLYBSD", runtime.newBoolean(os == OS.DRAGONFLYBSD));
```

Comment:
https://github.com/ffi/ffi/pull/724

Reply:
I nuked all but DragonFly from JRuby's `FFI::Platform` java code. Can nuke DragonFlyBSD once I see it in the ruby code.

Reply:
@headius `IS_DRAGONFLYBSD` is part of ffi's ruby code now.